src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int RANDOM = 30;
+    private static final int DEFAULT = 10;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getDefaultFigure() {
+        return new Circle(Color.WHITE, DEFAULT);
+    }
+
+    public Figure getRandomFigure() {
+        int randomize = random.nextInt(Color.values().length) + 1;
```

Comment:
why are we attached to color here?
use number 5 as a constant because you have 5 types of figures currently

Reply:
added constant